### PR TITLE
Lra 416 art survey filters and searches

### PR DESCRIPTION
**Pull request by britaumich**
here are filters with ransack gem

### CODE REVIEW

File: app/controllers/art_items_controller.rb [hunk 1]
```diff
@@ -63,8 +85,16 @@ def set_art_item
       @art_item = ArtItem.find(params[:id])
     end
 
+    def set_departments_list
```
Comment (rsmoke) [inline app/controllers/art_items_controller.rb:88]: dependency created ... shame on you.

File: app/controllers/art_items_controller.rb [hunk 2]
```diff
@@ -63,8 +85,16 @@ def set_art_item
...
+      @departments_list = Department.all
+    end
+
+    def set_appraisal_types
```
Comment (rsmoke) [inline app/controllers/art_items_controller.rb:92]: dependency created ... shame on you.

File: db/seeds.rb [hunk 3]
```diff
@@ -3,5 +3,18 @@
 #
```
Comment (rsmoke) [inline db/seeds.rb:3]: These seeds are all comments why keep them?

File: app/models/art_item.rb [hunk 4]
```diff
@@ -20,8 +20,16 @@ class ArtItem < ApplicationRecord
   belongs_to :department
   has_many :annotations
   has_rich_text :description
+  has_one :description, class_name: 'ActionText::RichText', as: :record
```
Comment (rsmoke) [inline app/models/art_item.rb:23]: What is this doing?

Comment (britaumich) [reply app/models/art_item.rb:23]: Because if you want to look at the application on your localhost you need to add these records to your database

Comment (britaumich) [reply app/models/art_item.rb:23]: You need this to run ransack search through ActionText fields (look at the VOD code)

**Review by rsmoke — APPROVED** [review]

**Pull request merged**